rosdep/base.yaml

```diff
@@ -4490,10 +4490,10 @@ libzip-dev:
 libzmq3-dev:
   arch: [zeromq]
   debian: [libzmq3-dev]
-  fedora: [zeromq-devel]
+  fedora: [cppzmq-devel]
```

Comment:
Does installing `cppzmq-devel` also mean `zeromq-devel` gets installed too? If so I think this is fine. If not then I think the rule should specify both `[cppzmq-devel, zeromq-devel]`

Reply:
> Does installing `cppzmq-devel` also mean `zeromq-devel` gets installed too?

Correct, `cppzmq-devel` can't be installed without `zeromq-devel`.